src/indexes.rs

```diff
@@ -763,30 +758,20 @@ impl Index {
     /// # let client = Client::new(MEILISEARCH_HOST, MEILISEARCH_API_KEY);
     /// # let index = client.create_index("get_tasks", None).await.unwrap().wait_for_completion(&client, None, None).await.unwrap().try_make_index(&client).unwrap();
     ///
-    /// let status = index.get_tasks().await.unwrap();
-    /// assert!(status.len() == 1); // the index was created
-    ///
-    /// index.set_ranking_rules(["wrong_ranking_rule"]).await.unwrap();
+    /// let tasks = index.get_tasks().await.unwrap();
     ///
-    /// let status = index.get_tasks().await.unwrap();
-    /// assert!(status.len() == 2);
+    /// assert!(tasks.results.len() > 0);
     /// # index.delete().await.unwrap().wait_for_completion(&client, None, None).await.unwrap();
     /// # });
     /// ```
-    pub async fn get_tasks(&self) -> Result<Vec<Task>, Error> {
-        #[derive(Deserialize)]
-        struct AllTasks {
-            results: Vec<Task>,
-        }
-
-        Ok(request::<(), AllTasks>(
-            &format!("{}/indexes/{}/tasks", self.client.host, self.uid),
+    pub async fn get_tasks(&self) -> Result<TasksResults, Error> {
+        request::<(), TasksResults>(
+            &format!("{}/tasks", self.client.host),
```

Comment:
I think this method should also have the `indexUid` right?
Something like:
```rust
    &format!("{}/tasks?indexUid={}",
        self.client.host,
        self.uid,
```

Reply:
Thanks a lot